temporian/implementation/numpy_cc/operators/window.cc

```diff
@@ -38,18 +38,36 @@ py::array_t<OUTPUT> accumulate(const ArrayD &evset_timestamps,
   size_t begin_idx = 0;
 
   for (size_t sampling_idx = 0; sampling_idx < n_event; sampling_idx++) {

```

Comment:
Nit: What about renaming "sampling_idx" with "end_idx" (more similar to begin_idx, more similar to the sampling based "accumulate" function.